assets/js/ajax-filters.js

```diff
@@ -122,7 +123,8 @@ jQuery( document ).ready( function ( $ ) {
 				if ( result ) {
 					try {
 						if ( result.showing ) {
-							$( showing ).show().html( '<span>' + result.showing + '</span>' + result.showing_links );
+							var showing_el = jQuery('<span>').text(result.showing)
+							$( showing ).show().html( '').text(result.showing_links).prepend(showing_el);
```

Comment:
`result.showing_links` is contains HTML and gets encoded using `$.text()`. 

<img width="578" alt="screen shot 2018-06-19 at 4 55 28 pm" src="https://user-images.githubusercontent.com/68693/41609571-09effaf2-73e3-11e8-95e4-a4399b7a0e51.png">